.node-version

```diff
@@ -0,0 +1 @@
+16.19.1
```

Comment:
```suggestion
16
```

I love the use of `.node-version`, but would prefer we lock major version, only.